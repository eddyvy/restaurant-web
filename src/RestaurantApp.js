import React from 'react'

export const RestaurantApp = () => {
    return (
        <div>
            <h1>Hola mundeteee!!!</h1>
        </div>
    )
}
