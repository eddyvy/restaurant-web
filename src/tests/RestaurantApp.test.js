import { shallow } from 'enzyme'

import { RestaurantApp } from '../RestaurantApp'


describe('Tets in <RestaurantApp />', () => {

    const wrapper = shallow( <RestaurantApp/> )
    
    test('should be shown correctly', () => {
        
        expect( wrapper ).toMatchSnapshot()

    })
    
})
