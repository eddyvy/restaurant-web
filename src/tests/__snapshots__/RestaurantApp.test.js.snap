// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tets in <RestaurantApp /> should be shown correctly 1`] = `
<div>
  <h1>
    Hola mundeteee!!!
  </h1>
</div>
`;
